rector.php

```diff
@@ -50,7 +57,7 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->sets([
         SetList::DEAD_CODE,
-        LevelSetList::UP_TO_PHP_74,
+        LevelSetList::UP_TO_PHP_80,
```

Comment:
it can use up to php 8.1 since php 8.0 no longer support on 4.5 branch

Reply:
Done.